gzll/CHANGELOG.rst

```diff
@@ -20,6 +20,11 @@ Added
 * Added the library variant for the nRF5340 network core.
 * Added DPPI to glue layer.
 
+Bug fixes
+=========
+
+* Fixed the :c:func:`nrf_gzll_init` function to avoid calling memcpy() with the same addresses.
+
```

Comment:
How about mentioning the workaround for errata 117? Errata 158 is not public yet, so probably should not be mentioned.

Reply:
Thanks